pkg/ingester/ingester.go

```diff
@@ -2238,8 +2242,11 @@ func (i *Ingester) getTSDBUsers() []string {
 }
 
 func (i *Ingester) getOrCreateTSDB(userID string, force bool) (*userTSDB, error) {
-	db := i.getTSDB(userID)
+	db, err := i.getTSDB(userID)
 	if db != nil {
+		if err != nil {
+			level.Warn(i.logger).Log("msg", "error getting user DB but userDB is not null", "err", err, "userID", userID)
```

Comment:
why would this happen ?

Reply:
it should never happen.. it does not change the fact that if it happens if good to have a log! maybe i can add a comment

Reply:
I was trying to understand if you had an edge case in mind. either  comment or 
```
if db != nil && err == nil
   return db, nil
```
works